src/utils/editor.test.jsx

```
@@ -16,7 +16,7 @@ import { unregisterDisallowedBlocks } from './blocks';
 vi.mock( '@wordpress/editor', () => ( {
 	store: { name: 'core/editor' },
 } ) );
-vi.mock( '@wordpress/data' );
+vi.mock( import( '@wordpress/data' ), { spy: true } );
```

Fix test failures by using the original module rather than a partial mock that caused errors within other `@wordpress` modules.